### PR TITLE
Feature/offline package with multiple websites

### PR DESCRIPTION
**Pull request by asiaboron**

### CODE REVIEW

**Review by schwindelig — APPROVED** [review]
Looks good to me, thank you. @ernstjoss do  you have any feedback?

File: src/templates/install.ps1 [hunk 1]
```diff
@@ -39,77 +80,59 @@ $role = $config.Role
...
     -Environment $environment `
     -Role $role `
     -BlueprintFolderPath $blueprintFolderPath `
     -ConfigFolder $configFolder
 
+StartAppPool $appPoolName
 
-Write-Output "Starting IIS app pool $appPoolName"
-Start-WebAppPool $appPoolName
+$additionalWebsites = $config.AdditionalWebsites.Website
```
Comment (ernstjoss) [inline src/templates/install.ps1:109]: Is this really right here? Is it not only "$config.AdditionalWebsites" which have the array of websites?

Comment (asiaboron) [reply src/templates/install.ps1:109]: Yes, I tested this :)
$config.AdditionalWebsites is a single object, $config.AdditionalWebsites.Website is a list.

**Review by ernstjoss — COMMENTED** [review]
Please add also a changelog for these enhancements pointing to the next version.

**Review by ernstjoss — APPROVED** [review]
Just asked one question, were I'm not quite sure.
Apart from that looks good to me as well.

**Pull request merged**